integration-tests/routes/search-test.js

```diff
@@ -207,4 +226,27 @@ describe('the search route', function() {
       });
     });
   });
+
+  describe('without javascript', function() {
+    it('renders search results', async function() {
+      await visit('/search/Salzburg', { disableJavascript: true }, async(page, $response) => {
+
+        await page.waitForSelector('[data-test-search-result="Salzburg"]');
+        await page.click('[data-test-search-result="Salzburg"] a');
+        await page.waitForSelector('[data-test-location]');
+  
+        expect(page.url()).to.have.path('/location/2');
+      });
+    });
```

Comment:
I think this doesn't actually test that search results are rendered but that navigating from search results to details page works? That is already covered by the [test above](https://github.com/simplabs/breethe-client/pull/92/files#diff-577670e6f179cf7aa04d01a8c75ee48eR5) so I think can just be removed.